_docs/sysadmin/configuration/ansible_course_creation.md

```diff
@@ -0,0 +1,36 @@
+---
+title: Course Creation
+category: System Administrator > Configuration & Administration
+redirect_from:
```

Comment:
The redirect is only needed when the page used to live under a different path (for backwards compabtibility).  Please remove these lines from the new files

Reply:
I removed the redirect line from the new file, ansible create course and term documentation.